src/transformers/file_utils.py

```diff
@@ -194,6 +194,40 @@ def is_faiss_available():
     return _faiss_available
 
 
+DATASETS_IMPORT_ERROR = """
+This {0} requires the 🤗 Datasets library but it was not found on your enviromnent. You can install it with:
+```
+pip install datasets
+```
+In a notebook or a colab, you can install it by executing a cell with
+```
+!pip install datasets
+```
+then restarting your kernel.
+
+Note that if you have a local folder named `datasets` or a local python file named `datasets.py` in your current
+working directory, python may try to import this instead of the 🤗 Datasets library. You should rename this folder or
+that python file if that's the case.
```

Comment:
```suggestion
Note that if you have a local folder named `datasets` with a `__init__.py` file inside or a local python file named `datasets.py` in your current working directory, python may try to import this instead of the 🤗 Datasets library. You should remove this `__init__.py` file or rename this folder or that python file if that's the case.
```

Reply:
One line is then unreadable in VS code :-(